index.js

```diff
@@ -164,6 +164,14 @@ function retryRequest(requestOpts, opts, callback) {
           streamResponseHandled = true;
           onResponse(null, resp, body);
         })
+        .on('data', data => {
+          if (streamResponseHandled) {
+            return;
+          }
+
+          streamResponseHandled = true;
+          onResponse(null, data);
```

Comment:
Is there a risk that more than one `data` event will be emitted on a large response?